File: acerte-ou-forca/acerte-ou-forca/Compartilha.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace acerte_ou_forca
{
    public class Compartilha
    {
        public enum Tema
        {
            Animais,
            Papelaria,
            Eletronicos
        }

        private static string[] palavras;
        private static int vitorias = 0;
        private static Tema tema;

        public string[] getPalavras()
        {
            return palavras;
        }
        
        public void setPalavras(string[] palavras2)
        {
            palavras = palavras2;
        }

        public int getVitorias()
        {
            return vitorias;
        }

        public void setVitorias()
        {
            vitorias++;
        }

        public Tema getTema()
        {
            return tema;
        }

        public void setTema(Tema tema2)
        {
            tema = tema2;
        }
    }
}

File: AOF_Java_version/src/com/acerteOuForca/Compartilha.java
package com.acerteOuForca;

import java.util.ArrayList;

public class Compartilha {

    public enum Tema{
        Animais,
        Eletronicos,
        Papelaria
    }
    private static ArrayList<String> palavras;
    private static int vitorias = 0;
    private static Tema tema;

    public ArrayList<String> getPalavras() {
        return palavras;
    }

    public void setPalavras(ArrayList<String> palavras) {
        Compartilha.palavras = palavras;
    }

    public int getVitorias() {
        return vitorias;
    }

    public void setVitorias() {
        Compartilha.vitorias++;
    }

    public Tema getTema() {
        return tema;
    }

    public void setTema(Tema tema) {
        Compartilha.tema = tema;
    }
}
